lib/backend/k8s/resources/profile.go

```diff
@@ -453,14 +434,10 @@ func (pw *profileWatcher) processProfileEvents() {
 		// Send the processed event.
 		select {
 		case pw.resultChan <- e:
-			// If this is an error event. check to see if it's a terminating one.
-			// If so, terminate this watcher.
+			// If this is an error event. terminate the watcher and bubble error to resenyc.
```

Comment:
Is this necessary now?  Can we not bubble up error events?

If the error is terminating a watcher then we'll see the channel close.  Can't we just bubble up the error without terminating?

Reply:
Sure, I will update to bubble up the error.